auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -116,14 +116,146 @@ def __call__(self, val_acc):
             self.counter = 0
 
 
+_libcudart = ctypes.CDLL("libcudart.so")
+# Set device limit on the current device
+# cudaLimitMaxL2FetchGranularity = 0x05
+p_value = ctypes.cast((ctypes.c_int * 1)(), ctypes.POINTER(ctypes.c_int))
+_libcudart.cudaDeviceSetLimit(ctypes.c_int(0x05), ctypes.c_int(128))
+_libcudart.cudaDeviceGetLimit(p_value, ctypes.c_int(0x05))
+assert p_value.contents.value == 128
+
+torch.backends.cudnn.benchmark = True
+torch.set_float32_matmul_precision("high")
+
+
+CONFIG = {
```

Comment:
This is duplicated

Reply:
I have removed any instances of duplication.